cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityWritePrivilegesTest.scala

```diff
@@ -59,6 +245,34 @@ class SecurityWritePrivilegesTest extends DocumentingTest with QueryStatisticsTe
             | See <<operations-manual#detach-delete-restricted-user, Operations Manual -> Fine-grained access control>> for more info.""".stripMargin)
       }
     }
+
+    section("`ALL GRAPH PRIVILEGES`", "administration-security-writes-all", "enterprise-edition") {
+      p(
+        """The `ALL GRAPH PRIVILEGES` privilege allows the user to execute any command on a graph.""".stripMargin)
+      p("include::grant-all-graph-privileges-syntax.asciidoc[]")
+
+      p(
+        """For example, granting all graph privileges on the graph `neo4j` to the role `regularUsers` would be achieved using:""".stripMargin)
+      query("GRANT ALL GRAPH PRIVILEGES ON GRAPH neo4j TO regularUsers", ResultAssertions(r => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        statsOnlyResultTable()
+      }
+      note {
+        p("Unlike the more specific read and write commands, it is not possible to restrict `ALL GRAPH PRIVILEGES` privileges to specific ELEMENTS, NODES or RELATIONSHIPS. If it is " +
+          "desirable to prevent a user from reading or writing to a subset of database objects, a `GRANT ALL GRAPH PRIVILEGES` can be combined with more specific DENYs targeting these elements.")
```

Comment:
```suggestion
          "desirable to prevent a user from reading or writing to a subset of database objects, a `GRANT ALL GRAPH PRIVILEGES` can be combined with more specific `DENY` commands to target these elements.")
```